docs/stencil-docs/installing-stencil-cli/incompatibile-directives.mdx

```diff
@@ -0,0 +1,74 @@
+# Incompatible directives
+
+Since we have different versions of node sass running in stencil cli and production, there can be difference in the output of the css.
```

Comment:
```suggestion
Since we have different versions of node-sass running in Stencil CLI and production, there can be differences in the output of the CSS.
```